src/core_plugins/kibana/public/discover/embeddable/search_embeddable_handler.js

```diff
@@ -0,0 +1,77 @@
+import searchTemplate from './search_template.html';
+import angular from 'angular';
+import * as columnActions from 'ui/doc_table/actions/columns';
+import { getPersistedStateId } from 'plugins/kibana/dashboard/panel/panel_state';
+import { EmbeddableHandler } from 'ui/embeddable';
+
+
+export class SearchEmbeddableHandler extends EmbeddableHandler {
+
+  constructor($compile, $rootScope, searchLoader) {
+    super();
+    this.$compile = $compile;
+    this.searchLoader = searchLoader;
+    this.$rootScope = $rootScope;
+    this.name = 'search';
+  }
+
+  async getEditPath(panelId) {
+    return await this.searchLoader.urlFor(panelId);
```

Comment:
Yea, that's a good point. Not sure how I feel about adding that functionality now though when it's not needed by anything we build.  Yes, we want this to be a flexible embeddable system for others to easily write panels in, but I think during this initial phase, it's okay to limit what the container api exposes.   We should probably add some kind of forceRefresh functionality that forces a rebuild of the whole panel at some point. Just not sure I want to go down this path without an easy way to test it works.  Thoughts?